telemetry/vscode/src/generateTelemetry.ts

```diff
@@ -50,14 +51,20 @@ function formatOutput(output: string) {
     `
 
     const args = parseArguments()
-    const input: MetricDefinitionRoot = args.inputFiles.map(validateInput).reduce(
+    const rawDefinitions: MetricDefinitionRoot = args.inputFiles.map(validateInput).reduce(
         (item: MetricDefinitionRoot, input: MetricDefinitionRoot) => {
             item.types!.push(...(input.types ?? []))
             item.metrics.push(...input.metrics)
             return item
         },
         { types: [], metrics: [] }
     )
+    // Allow read in files to overwrite default definitions. First one wins, so the extra
```

Comment:
It would be nice to see a test for TS similar to `test/kotlin/software/aws/toolkits/telemetry/generator/GeneratorTest.kt` - there doesn't appear to be any validation around this change.

Reply:
That is outside of the scope of this PR, there are no tests for the Typescript generator (we have an item in the backlog for it)